changelogs/fragments/2056-zos_job-modules-adding-cpu_time-execution_node-origin_node.yml

```
@@ -0,0 +1,9 @@
+minor_changes:
+   - zos_job_submit -  Adds new response parameters - cpu_time, origin_node and execution_node.
```

```suggestion
   - zos_job_submit - Adds new fields cpu_time, origin_node and execution_node to response.
```

---

Will make another suggestion, these are known as return values.

```suggestion
   - zos_job_submit -  Adds new return values cpu_time, origin_node and execution_node to response.
```